src/Config.hs

```diff
@@ -41,13 +43,26 @@ type App = AppT IO
 -- running in and a Persistent 'ConnectionPool'.
 data Config
     = Config
-    { getPool :: ConnectionPool
-    , getEnv  :: Environment
-    , getMetrics :: Metrics
+    { getPool      :: ConnectionPool
+    , getEnv       :: Environment
+    , getMetrics   :: Metrics
+    , configLogEnv :: LogEnv
```

Comment:
I do prefer the new naming convention for the config's fields. However, for consistency's sake, if we're going to change them, then we should change all of them at once.